electron/legendary/games.ts

```diff
@@ -311,7 +333,14 @@ class LegendaryGame extends Game {
       autoInstallDxvk
     } = await this.getSettings()
 
+    const { discordrpc } = (await GlobalConfig.get().getSettings())
+
     if (isWindows) {
+
+      if (discordrpc) {
```

Comment:
remove this if statement. Because you don't need to call the same thing again. Just call it once after you got the discordrpc value. It does not rely on the platform.